test/process.jl

```diff
@@ -1,3 +1,27 @@
 @testset "process.jl" begin
-    println("No Tests written for process.jl")
+
+    @process helloWorld() begin
+        return "Goodbye world"
+    end
+
+    @test helloWorld().name == "helloWorld()"
+    # Still need to test out the Task attribute and all the other ones, although some will be null
+
+    @testset "Process Sig Error" begin
+        try
+            eval(:(@process 4 :4))  # first arg is wrong
+        catch e
+            @test typeof(e.error) === ArgumentError
+        end
+
+    end
+
+    # This testset isn't running... why?
```

Comment:
I would guess that the test set is running but your @test is within the catch which is never triggering because there is never an error thrown. Isn't this the @capture issue that we were talking about last Senior Design?

Reply:
The thing is that's it's not passing or failing. There are 7 tests before this testset, and commenting it out or adding it does not increase the amount. If I comment out the other test set then the counter is decreased like expected, so it doesn't seem to pass, it seems to not run. Not sure though, but that was what I observed

Reply:
I just tested it and it is because the thing is never being thrown. If I place a failing test anywhere else it fails so the test set works. The catch block is just never triggering.